README.md

```diff
@@ -17,63 +17,43 @@
 |Original Lyrics | Replacement Lyrics|
 |--------------------------- | ---------------------------|
 |There once was a ship that put to sea<br>And the name of that ship was the Billy o' Tea<br>The winds blew hard, her bow dipped down<br>Blow, me bully boys, blow (huh)|There once was a class, a sight to see<br>the twenty eight cohort RPP<br>The sprints were hard, some got them down<br>code, me problem toys, code (huh)|
+
+|Soon may the Wellerman come<br>To bring us sugar and tea and rum<br>One day, when the tonguing' is done<br>We'll take our leave and go|LinkedIn decides our fate<br>Kitchen time, don't be late!<br>Push code, verbose mode<br>We're cohort twenty eight!|
+
+
+|She had not been two weeks from shore<br>When down on her a right whale bore<br>The captain called all hands and swore<br>He'd take that whale in tow (huh)| Our lives had been such bores<br>When we came to explore<br>Precourse came and opened the door<br>To a world of so much mo'<br>|
+
+|Soon may the Wellerman come<br>To bring us sugar and tea and rum<br>One day, when the tonguing' is done<br>We'll take our leave and go|LinkedIn decides our fate<br>Kitchen time, don't be late!<br>Push code, verbose mode<br>We're cohort twenty eight!|
+
+> Before the boat had hit the water
+> The whale's tail came up and caught her
+> All hands to the side, harpooned and fought her
+> When she dived down below (huh)
+
+|Soon may the Wellerman come<br>To bring us sugar and tea and rum<br>One day, when the tonguing' is done<br>We'll take our leave and go|LinkedIn decides our fate<br>Kitchen time, don't be late!<br>Push code, verbose mode<br>We're cohort twenty eight!|
```

Comment:
Thanks for adding the other chorues! Please feel free to write another verse, or tweek the chorus. I would like to match the "end-rhyming" of the original if possible, but I wanted to end with the "cohort twenty eight!"

end-rhyme of original chorus:
****** Come
****** Rum
****** Done
****** Go